ads/google/a4a/docs/RTB-Implementation.md

```diff
@@ -0,0 +1,65 @@
+# AMP Ad Implementation Guide for RTB Ad Exchanges
+---
+## Objective
+ 
+This guide is designed to provide additional information for SSPs and Ad Exchanges that want to support AMP Ads in a Real-Time Bidding (RTB) environment.  The IAB's OpenRTB 2.5 spec is [here](http://www.iab.com/wp-content/uploads/2016/03/OpenRTB-API-Specification-Version-2-5-FINAL.pdf&sa=D&ust=1496436943725000&usg=AFQjCNFlig47imweXbZx-yuvW8LmlonzFw).
+ 
+For AMP Ads on AMP Pages, a major benefit is that if the ad response is AMP, then the ad can be rendered early by splicing it into the surrounding AMP page, with no need for iframing or delays, without risk to the page's UX.  The AMP page needs to protect itself from non-AMP content, so must delay the rendering of any ad that has not been blessed by trusted AMP verification.
+ 
+# Background Docs
```

Comment:
I suggest:

1.  Make this a heading level 2.
2. Move this section to the bottom of the document, where typical reference docs are.